docs/source/Install/installdependencies.md

```diff
@@ -113,25 +113,27 @@ If you run Pulsar in Docker, you will now be able to see your container in the d
 
 Everything should now be installed and ready for us to get your first Raphtory Job underway!
 
-## Downloading the latest Raphtory release
-To begin, you must clone the example repository from [Github](https://github.com/Raphtory/Examples.git) and download the latest build of [Raphtory 0.5.0-pometry-release)](https://github.com/Raphtory/Raphtory/releases/tag/0.5.0-alpha). 
+## Running the latest Raphtory release in an Examples Project
+All the example projects can be found in the [Raphtory repo](https://github.com/Raphtory/Raphtory/tree/development), currently the projects in the `development` branch are working (raphtory-example-presto is in progress).
```

Comment:
Can you just change this to say currently all projects are working other than presto which will be re-enabled soon.

This way we don't have to update when we merge into master :)